contracts/modules/commons/ModuleAuth.sol

```diff
@@ -86,7 +86,7 @@ abstract contract ModuleAuth is IModuleAuth, ModuleERC165, SignatureValidator, I
       imageHash = keccak256(abi.encode(imageHash, addrWeight, addr));
     }
 
-    return totalWeight >= threshold && _isValidImage(imageHash);
+    return imageHash != bytes32(0) && totalWeight >= threshold && _isValidImage(imageHash);
```

Comment:
Might be better to put this `imageHash != bytes32(0)` check within `_isValidImage()` since it could be called by various methods.